tests/unit/job.spec.ts

```diff
@@ -33,7 +25,7 @@ describe('# JOB', () => {
       canceled: true,
       stdLotSize: 1,
     };
-    await CLIENT.db().collection('jobs').insertOne(doc);
+    await TEST_DB_CLIENT.db().collection( 'jobs' ).insertOne(doc);
```

Comment:
Might be easier to define a variable JOB_COLLECTION and set it to `JobModel.collection.name` and replace the `jobs` text throughout....just in case we ever change the name.